lib/shopify_app/sessions_concern.rb

```diff
@@ -49,6 +50,8 @@ def login_shop
       sess = ShopifyAPI::Session.new(shop_name, token)
       session[:shopify] = ShopifyApp::SessionRepository.store(sess)
       session[:shopify_domain] = shop_name
+
+      perform_after_install_actions
```

Comment:
I'm pretty sure this gets called even if the shop is logging in for a second time. The only way to do a true `after_install` is to check if the is creating the shop record but this assumes certain things about the SessionRepository. My opinion has always been that its better to keep it simple and use after login hooks but ensure that any actions performed are idempotent. This is how the WebhooksManager and ScriptTagsManager currently function.

All that is to say - lets just have the one `perform_after_authenticate_actions`